service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ManageOrdersAboutToSubmitControllerTest.java

```diff
@@ -123,7 +123,7 @@ void mocks() {
     void shouldBuildNewOrderObject() {
         CaseData caseData = buildCaseData().toBuilder()
             .manageOrdersEventData(ManageOrdersEventData.builder()
-                .manageOrdersType(C32_CARE_ORDER)
+                .manageOrdersType(C21_BLANK_ORDER)
```

Comment:
For my own curiosity, is there any reason you had to change the order type here?

Reply:
If we keep it as Care Order then the children would need to have their "finalOrderIssued" field updated to be "Yes" and would need to change the CHILDREN variable but that variable is also used elsewhere. 

Seemed like it would be less messy to just change the order type rather than create another variable or a function since the test is just checking an order object can be built